crates/polars/tests/it/core/joins.rs

```diff
@@ -614,3 +614,503 @@ fn test_4_threads_bit_offset() -> PolarsResult<()> {
     assert_eq!(out.shape(), (1, 2));
     Ok(())
 }
+
```

Comment:
Can you remove those tests? We want to do as much tests in python as possible as our CI is slow enough as it is.